src/models/onnxruntime_api.h

```diff
@@ -144,6 +129,31 @@ inline std::string GetCurrentModuleDir() {
   return module_directory;
 }
 
+inline void* LoadDynamicLibraryIfExists(const std::string& path) {
+  LOG_INFO("Attempting to dlopen %s", path.c_str());
+  void* ort_lib_handle = dlopen(path.c_str(), RTLD_NOW | RTLD_LOCAL);
+  if (ort_lib_handle == nullptr) {
+    char* err = dlerror();
+    LOG_WARN("Error while dlopen: %s", (err != nullptr ? err : "Unknown"));
+    // Trying current dir
```

Comment:
This is needed for NuGet on Linux, because `dlopen` won't search the current directory.

Reply:
My understanding is that this should typically be resolved with LD_LIBRARY_PATH. Is that not what is recommended?

Reply:
Do you mean we add instructions for user to tweak `LD_LIBRARY_PATH`?

In #731 you managed to load the ORT library using `ctypes` in Python. Python has this global location for site-packages. But for NuGet & C# projects, dependencies are copied and stored locally in the artifact folder. Therefore, I think it would be better for us to just load the nearby libraries, because this *should* be the desired library.

Reply:
ok, if it is guaranteed that the ort lib will end up in the same folder as ortgenai lib, this should be a good strategy.